_conferences/devfest-mn-2018.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "DevFestMN"
```

Comment:
This should be "GDG DevFest" to match the others.

Reply:
Thanks for the heads up